test/gpc.js

```diff
@@ -0,0 +1,62 @@
+const assert = require('assert')
+
+const {
+    GPC_HEADER_PRIORITY
+} = require('../lib/rulePriorities')
+
+const {
+    generateGPCheaderRules
+} = require('../lib/gpc')
+
+const baseExtensionConfig = {
+    features: {
+        gpc: {
+            state: 'enabled',
+            exceptions: [{
+                domain: 'content-blocking1.example',
+                reason: '1st GPC header  reason'
+            }, {
+                domain: 'content-blocking2.example',
+                reason: '2nd GPC header  reason'
+            }]
+        }
+    }
+}
+
+const expectedGPCRule = [{
```

Comment:
Maybe call it result, since it includes the lookup too?
```suggestion
const expectedGPCResult = [{
```